apps/omg_api/lib/block_queue.ex

```diff
@@ -118,6 +119,14 @@ defmodule OMG.API.BlockQueue do
 
       interval = Application.fetch_env!(:omg_api, :block_queue_eth_height_check_interval_ms)
       {:ok, _} = :timer.send_interval(interval, self(), :check_ethereum_status)
+      parent = self()
+
+      {:ok, _} =
+        Recorder.start_link(%Recorder{
```

Comment:
is doing `.start_link` safe for a seemingly non-critical service? Do we want the recorded process to go down when the `Recorder` does and vice versa?

Reply:
The operation that Recorder does is not complex and failures are preventable - but it does expect the parent (in this example BlockQueue) to be running and provide his pid. As links are bidirectional the only other way would be to keep the Recorder pid in state and forcefully cleanup in terminate.

Reply:
There's a third way. We keep the functionality in the process and don't have the Recorderer at all.

Reply:
gotcha, let's leave as is then :+1: